test/SlowTests/Server/Documents/SqlMigration/SqlAwareTestBase.cs

```diff
@@ -207,30 +208,37 @@ private DisposableAction WithMsSqlDatabase(out string connectionString, out stri
 
         protected DisposableAction WithMySqlDatabase(out string connectionString, out string databaseName, string dataSet, bool includeData = true)
         {
+            const int commandTimeout = 2 * 60;
```

Comment:
I checked it and because of all the SQL database run on the same machine, the requests can take time and lead to timeout exception. I set it to two minutes to be sure it has its time